docs/source/developer_docs.rst

```diff
@@ -7,7 +7,7 @@ The Jupyter widgets packages are developed in the `https://github.com/jupyter-wi
 Scope of ipywidgets
 -------------------
 
-``ipywidgets`` is a framework to provide eventful python objects that have a representation in the browser (see :doc:`examples/Widget Basics.ipynb` for more on the definition of widgets). This requires two components:
+``ipywidgets`` is a framework to provide eventful python objects that have a representation in the browser (see "Simple Widget Introduction" in the :doc:`user_guide` for more on the definition of widgets). This requires two components:
```

Comment:
You can simply drop the file extension to fix the link.

Sphinx document names don't include a file suffix.

http://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-doc

Reply:
Thanks; several of your fixes in #2660 are more elegant than what is in here...